slam_toolbox/src/slam_toolbox_common.cpp

```diff
@@ -446,10 +443,11 @@ bool SlamToolbox::shouldProcessScan(
   }
 
   // check moved enough, within 10% for correction error
-  const double dist2 = fabs((last_pose.GetX() - pose.GetX())*(last_pose.GetX() - 
-    pose.GetX()) + (last_pose.GetY() - pose.GetY())*
-    (last_pose.GetX() - pose.GetY()));
-  if(dist2 < 0.8 * min_dist2 || scan->header.seq < 5)
+  const auto sq_dist_to_last_accepted_pose = last_pose.SquaredDistance(pose);
```

Comment:
Remove auto from both lines, that's not in style for this codebase

Reply:
fixed